### DISCUSSION TITLE
Does latest ROCm 6.0  support Radeon 680M（gfx1035） ?

### DISCUSSION THREAD

**Discussion by PennyFranklin** [ROCm/TheRock#8002] [Q&A]
Does latest ROCm 6.0 support Radeon 780M ? This chip is part of mobile cpu Ryzen7 6800H.
if its not supported - is there any plans to add support of this GPU?
I have a machine learning class that need to install Anaconda and PyTorch, I wonder if I can explore it on ubuntu or I can only install the CPU version on win11.

**Comment by rafalgajda-silvair** [comment]
I have the latest version installed on Ubuntu and looking through the files I don't see any for `gfx1035` I'm afraid. 
I have the Radeon 780M which I hoped to use for some AI experiments and it isn't supported either :(
Looks like iGPU-s are lower/none priority which is too bad cause there would be perfect as an entry point to ML.

You should be able to run pytorch on the CPU on Ubuntu so at least you don't have to suffer with Windows ;)

**Reply by PennyFranklin** [reply]
So sad to hear that:(

**Reply by toucankd** [reply]
Actually Radeon 680M and 780M are supported by the latest ROCm 6.0, what you need to do is to set **HSA_OVERRIDE_GFX_VERSION=10.3.0** for 680M, and **HSA_OVERRIDE_GFX_VERSION=11.0.0** for 780M.

**Reply by BikeHelmet** [reply]
It has been patched to the point that they don't work anymore, but there is a side project that keeps the original support.
https://github.com/likelovewant/ollama-for-amd/releases

After fighting with it for hours, I stumbled across it and everything was ticky boo afterwards. (Radeon 680M / gfx1035)

**Reply by v1ckxy** [reply]
@BikeHelmet If it's true that they have patched that so that it doesn't work on the 680m/780m, then they have no shame.

**Reply by robertojguerra** [reply]
Besides ollama for AMD, is this another project that brings rocm to more AMD chips?
https://github.com/rocm/therock

**Reply by robertojguerra** [reply]
I had some success with ROCm 7.14
https://github.com/ROCm/TheRock/discussions/7199

Do you think that https://github.com/likelovewant/ollama-for-amd/releases would be better?

**Reply by wizardwithcodehazard** [reply]
Try using vulkun

**Comment by harkgill-amd** [comment]
While gfx1035 isn't officially supported, you can find nightly builds for it at https://nightly.repo.amd.com/rocm/whl-next/. Instructions on how to get up and running with those builds https://github.com/ROCm/TheRock/blob/main/RELEASES.md.

**Reply by robertojguerra** [reply]
Specific instructions for comfyUI
https://github.com/ROCm/TheRock/discussions/7199
